README.md

```diff
@@ -37,21 +36,21 @@ infer 5000 Churn
 
 You can add flexibility to the training and inference processes using additional hyperparameters.
 
-`train PATH_TO_ORIGINAL_CSV –metadata_path PATH_TO_METADATA_JSON –table_name TABLE_NAME –epochs INT –row_limit INT –dropna BOOL`
+`train --path PATH_TO_ORIGINAL_CSV --metadata_path PATH_TO_METADATA_JSON --table_name TABLE_NAME --epochs INT --row_limit INT --drop_null BOOL`
```

Comment:
Please, remove here --metadata_path because you has already used --metadata_path.
Also it should be better to rename PATH_TO_METADATA_JSON to PATH_TO_METADATA_YAML